cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -1273,6 +1276,76 @@ public void testUpdateSchedulesFlag() throws Exception {
     Assert.assertEquals(2, actualSchedules.size());
   }
 
+  @Test
+  public void testAddScheduleWithDisabledProfile() throws Exception {
```

Comment:
this should be tested at a lower level than through the http handler. Putting it here will make the test take a long time, and also test a bunch of things that are not related to schedules and profiles. Can you put it in the CoreSchedulerService test?

Reply:
moved to CoreSchedulerServiceTest